crates/sdk/src/tx.rs

```diff
@@ -420,11 +420,9 @@ pub fn display_batch_resp(context: &impl Namada, resp: &TxResponse) {
             InnerTxResult::Success(_) => {
                 display_line!(
                     context.io(),
-                    "Transaction {} was successfully applied at height {}, \
-                     consuming {} gas units.",
+                    "Transaction {} was successfully applied at height {}.",
```

Comment:
why did you decide to remove the gas consumption of the tx in the logs?

Reply:
My bad, brought it back in 209da0c6ad